### PR TITLE
reduced numbers of pylint warnings

### PR DESCRIPTION
**Pull request by kleinicke**
Reduced the number of pylint warnings. I hope, I didn't break anything (for me everything works as before)

### CODE REVIEW

File: glasses/models/AutoTransform.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import torch
 import torchvision.transforms as T
-from glasses.models import *
```
Comment (kleinicke) [inline glasses/models/AutoTransform.py:3]: I don't know what this import was important for.

**Comment by kleinicke** [comment]
What does the codecov score exactly mean? 
Replacing a list as the default value by none
```
if num_pools is None:
    num_pools = [1, 4, 16]
```
decreased the score of the file SpatialPyramidPool.py by 5%.
Is this because num_pools is defined in every test, so the default value is never used?

**Comment by FrancescoSaverioZuppichini** [comment]
@kleinicke I think you are right

**Comment by FrancescoSaverioZuppichini** [comment]
@kleinicke thanks for working on this

**Pull request merged**